src/platform/export/exportToPDF.ts

```diff
@@ -0,0 +1,24 @@
+import { inject, injectable } from 'inversify';
+import { CancellationToken, NotebookDocument, Uri } from 'vscode';
+import { PythonEnvironment } from '../pythonEnvironments/info';
+import { ExportFormat, IExportBase, INbConvertExport } from './types';
+
+@injectable()
+export class ExportToPDF implements INbConvertExport {
```

Comment:
It looks like all of your renames are not be detecting as renames? I usually force this with `git add -A` but not sure if you can do it after the fact.

Reply:
didn't know I could do that, will try with next rename 💪